models/aeif_psc_alpha.cpp

```diff
@@ -94,19 +94,20 @@ nest::aeif_psc_alpha_dynamics( double,
   // good compiler will optimize the verbosity away ...
 
   // shorthand for state variables
-  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bound V
+  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bind V to the
```

Comment:
I think this comment relates to an outdated version, current comment is
``// bind V to the USER DEFINED V_peak_ value in Parameters.``